storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/resolvers/UserWithTweetsGetResolver.java

```diff
@@ -0,0 +1,82 @@
+package com.pushtorefresh.storio.sample.db.resolvers;
+
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+
+import com.pushtorefresh.storio.sample.db.entities.Tweet;
+import com.pushtorefresh.storio.sample.db.entities.User;
+import com.pushtorefresh.storio.sample.db.entities.UserWithTweets;
+import com.pushtorefresh.storio.sample.db.tables.TweetsTable;
+import com.pushtorefresh.storio.sqlite.StorIOSQLite;
+import com.pushtorefresh.storio.sqlite.operations.get.GetResolver;
+import com.pushtorefresh.storio.sqlite.queries.Query;
+import com.pushtorefresh.storio.sqlite.queries.RawQuery;
+
+import java.util.List;
+
+public final class UserWithTweetsGetResolver extends GetResolver<UserWithTweets> {
+
+    // We can even reuse existing get resolvers for our needs
+    // But, you can always write custom code, of course.
+    @NonNull
+    private final GetResolver<User> userGetResolver;
+
+    // Sorry for this hack :(
+    // We will pass you an instance of StorIO
+    // into the mapFromCursor() in v2.0.0.
+    //
+    // At the moment, you can save this instance in performGet() and then null it at the end
+    @NonNull
+    private final ThreadLocal<StorIOSQLite> storIOSQLiteFromPerformGet = new ThreadLocal<StorIOSQLite>();
```

Comment:
As soon as we will complete all issues with mileston v2.0.0 :) No concrete dates, but probably near to the end of this year, I hope :)